hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -29,12 +54,41 @@ def lambda_handler(event, context):
     an action message on the actions queue and here's where they're popped
     off and dealt with.
     """
+    lambda_init_once()
     for sqs_record in event["Records"]:
         # TODO research max # sqs records / lambda_handler invocation
-        action_message = ActionMessage.from_aws_message(json.loads(sqs_record["body"]))
+        action_message = ActionMessage.from_aws_message(sqs_record["body"])
```

Comment:
not json anymore I take it?

Reply:
The `from_aws_message` includes json.loads